pkgs/os-specific/linux/alsa-lib/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./alsa-plugin-conf-multilib.patch
+    (fetchpatch {
+      # plucked from upstream master, delete in next release
```

Comment:
Please comment why we need this rather then from where you got it which we can easily see by the link.

Reply:
fair point! added an additional comment about the patch's purpose. PTAL